src/Components/Blazor/BlazorExtension/src/Microsoft.VisualStudio.BlazorExtension.csproj

```diff
@@ -101,174 +98,50 @@
+      <Link>Content\LICENSE.txt</Link>
+    </Content>
+
+    <None Include="source.extension.vsixmanifest" SubType="Designer" />
   </ItemGroup>
-  <!--
-    We need to generate the assembly attributes for our assembly using the version from the build, so
-    we can flow it to the about dialog.
-  -->
+
   <ItemGroup>
-    <_VSIXAssemblyAttribute Include="System.Reflection.AssemblyInformationalVersionAttribute">
-      <_Parameter1>$(VersionPrefix)-$(VersionSuffix)</_Parameter1>
-    </_VSIXAssemblyAttribute>
+    <ProjectReference Include="..\..\Templates\src\Microsoft.AspNetCore.Blazor.Templates.csproj" />
   </ItemGroup>
-  <Target Name="_GenerateVSIXAssemblyAttributesHash" DependsOnTargets="PrepareForBuild" Condition="'@(_VSIXAssemblyAttribute)' != ''">
-    <!-- We only use up to _Parameter1 for most attributes, but other targets may add additional assembly attributes with multiple parameters. -->
-    <Hash ItemsToHash="@(_VSIXAssemblyAttribute->'%(Identity)%(_Parameter1)%(_Parameter2)%(_Parameter3)%(_Parameter4)%(_Parameter5)%(_Parameter6)%(_Parameter7)%(_Parameter8)')">
-      <Output TaskParameter="HashResult" PropertyName="_VSIXAssemblyAttributesHash" />
-    </Hash>
-    <WriteLinesToFile Lines="$(_VSIXAssemblyAttributesHash)" File="$(_GeneratedVSIXAssemblyInfoInputsCacheFile)" Overwrite="True" WriteOnlyWhenDifferent="True" />
-    <ItemGroup>
-      <FileWrites Include="$(_GeneratedVSIXAssemblyInfoInputsCacheFile)" />
-    </ItemGroup>
-  </Target>
-  <Target Name="_GenerateVSIXAssemblyAttributes" DependsOnTargets="_GenerateVSIXAssemblyAttributesHash" Inputs="$(_GeneratedVSIXAssemblyInfoInputsCacheFile)" Outputs="$(_GeneratedVSIXAssemblyInfoFile)" BeforeTargets="CoreCompile">
-    <ItemGroup>
-      <Compile Include="$(_GeneratedVSIXAssemblyInfoFile)">
-        <Visible>false</Visible>
-      </Compile>
-    </ItemGroup>
-    <WriteCodeFragment AssemblyAttributes="@(_VSIXAssemblyAttribute)" Language="C#" OutputFile="$(_GeneratedVSIXAssemblyInfoFile)" />
-    <ItemGroup>
-      <FileWrites Include="$(_GeneratedVSIXAssemblyInfoFile)" />
-    </ItemGroup>
-  </Target>
-  <!--
-
-    END INTERESTING STUFF
 
-  -->
-  <PropertyGroup>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectTypeGuids>{82b43b9b-a64c-4715-b499-d71e9ca2bd60};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <ProjectGuid>{9088E4E4-B855-457F-AE9E-D86709A5E1F4}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Microsoft.VisualStudio.BlazorExtension</RootNamespace>
-    <AssemblyName>Microsoft.VisualStudio.BlazorExtension</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
-    <GeneratePkgDefFile>true</GeneratePkgDefFile>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
```

Comment:
Not 100% sure, but I think this Microsoft.NET.Sdk sets this to portable. Not sure if that's important or not.

Reply:
Checked the Razor vsix, and it produces portable pdbs:

![image](https://user-images.githubusercontent.com/174281/52820228-4985a600-3060-11e9-8165-e782d2523ec0.png)

Guess it doesn't matter.